django_app/django_app/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for django_app project.
+
+Generated by 'django-admin startproject' using Django 4.0.
```

Comment:
These template messages can be removed and the links should be documented elsewhere.